### ISSUE TITLE
Remove X11 keyboard support

### ISSUE THREAD

**Issue by baka0815** [reicast/reicast-emulator#1287]
As Linux supports the keyboard as an input device, we might as well remove the legacy X11 keyboard support. One reason would be that it can confuse people if activated (see #1035).

There is also currently no way to configure the X11 keyboard as the inputs are all hard coded.
Or is there any reason to still support this? Then we should enable the configuration of the X11 keyboard.

**Comment by AbandonedCart** [comment]
🤷‍♀️

**Comment by Yrij-Zhavoronkov** [comment]
We can rewrite this))  For e.x. load keycodes from config.keyboard )))

**Comment by AbandonedCart** [comment]
@Yrij-Zhavoronkov I think you are talking about the hardware keyboard, which would replace x11

**Comment by Yrij-Zhavoronkov** [comment]
yes, you right. I talking about hardware keyboard (via eventX)

**Comment by Yrij-Zhavoronkov** [comment]
but how you can use keyboard via eventX - it needed root privs ;))   
x11 keyoboard work without it

**Comment by baka0815** [comment]
You can use the keyboard via evdev (without root).

**Comment by Yrij-Zhavoronkov** [comment]
good) but how? I can't listed it without it via 'evtest'

**Comment by baka0815** [comment]
That's odd, my keyboard is visible to ``evtest``. Are other devices like mice and gamepads listet by ``evtest``? Maybe you need to pe part of a specific group? Does ``sudo evtest`` list your device?

**Comment by Yrij-Zhavoronkov** [comment]
Are other devices like mice and gamepads listet by evtest? -Yes (as user)
Does sudo evtest list your device? -yes. 
but sudo != user )

**Comment by Yrij-Zhavoronkov** [comment]
proof:
evtest
```
No device specified, trying to scan all of /dev/input/event*
Not running as root, no devices may be available.
Available devices:
/dev/input/event14:	USB Controller
Select the device event number [0-14]:
```
sudo evtest
`No device specified, trying to scan all of /dev/input/event*
Available devices:
/dev/input/event0:	AT Translated Set 2 keyboard
/dev/input/event1:	SynPS/2 Synaptics TouchPad
/dev/input/event2:	RAPOO RAPOO 2.4G Wireless Device
/dev/input/event3:	RAPOO RAPOO 2.4G Wireless Device
/dev/input/event4:	Lid Switch
/dev/input/event5:	Sleep Button
/dev/input/event6:	Power Button
/dev/input/event7:	Video Bus
/dev/input/event8:	Video Bus
/dev/input/event9:	Asus WMI hotkeys
/dev/input/event10:	USB 2.0 UVC VGA WebCam: USB2.0 
/dev/input/event11:	HDA Intel PCH Mic
/dev/input/event12:	HDA Intel PCH Headphone
/dev/input/event13:	HDA Intel PCH HDMI/DP,pcm=3
/dev/input/event14:	USB Controller
Select the device event number [0-14]:`

**Comment by baka0815** [comment]
What does ``ls -la /dev/input/`` show? Does ``event0`` have other permissions that ``event14``?

**Comment by Yrij-Zhavoronkov** [comment]
```
crw-rw----   1 root input 13, 64 Aug 24 22:22 event0
crw-rw----   1 root input 13, 65 Aug 24 22:22 event1
crw-rw----   1 root input 13, 74 Aug 24 22:22 event10
crw-rw----   1 root input 13, 75 Aug 24 22:22 event11
crw-rw----   1 root input 13, 76 Aug 24 22:22 event12
crw-rw----   1 root input 13, 77 Aug 24 22:22 event13
crw-rw----+  1 root input 13, 78 Aug 24 22:59 event14
crw-rw----   1 root input 13, 66 Aug 24 22:36 event2
crw-rw----   1 root input 13, 67 Aug 24 22:36 event3
crw-rw----   1 root input 13, 68 Aug 24 22:22 event4
crw-rw----   1 root input 13, 69 Aug 24 22:22 event5
crw-rw----   1 root input 13, 70 Aug 24 22:22 event6
crw-rw----   1 root input 13, 71 Aug 24 22:22 event7
crw-rw----   1 root input 13, 72 Aug 24 22:22 event8
crw-rw----   1 root input 13, 73 Aug 24 22:22 event9
crw-rw-r--+  1 root input 13,  0 Aug 24 22:59 js0

```

**Comment by baka0815** [comment]
Yes, event14 has special privileges, however I don't know how to change that.
You should, however, add your user to the group ``input``, that should be enough as the group als has ``rw`` rights.
You need to restart your session afterwards (log off and on again) and you can check your groups via the ``groups`` command.
